src/main/java/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,64 @@
+package bookstore.controller;
+
+import bookstore.dto.BookDto;
+import bookstore.dto.BookSearchParametersDto;
+import bookstore.dto.CreateBookRequestDto;
+import bookstore.service.BookService;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/books")
+public class BookController {
+    private final BookService bookService;
+
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    @GetMapping
+    public List<BookDto> getAll() {
+        return bookService.findAll();
+    }
+
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    @GetMapping("/{id}")
+    public BookDto getBookById(@PathVariable Long id) {
```

Comment:
you also can add @validated annotation on your controller class in order to make validation of @PathVariable, @RequestParam etc. (optional)

```suggestion
    public BookDto getBookById(@PathVariable @Positive Long id) {
```